indy_node/test/write_permission/test_revocation_write_permission.py

```diff
@@ -5,57 +5,160 @@
+    del rev_reg_entry[VALUE][PREV_ACCUM]
+    rev_entry_req_client = sdk_sign_request_from_dict(looper, sdk_wallet_client, rev_reg_entry)
+    with pytest.raises(RequestRejectedException):
+        sdk_send_and_check([json.dumps(rev_entry_req_client)], looper, txnPoolNodeSet, sdk_pool_handle)
+
+
+def test_trust_anchor_not_owner_cant_create_revoc_reg_entry(looper,
+                                                            txnPoolNodeSet,
+                                                            sdk_wallet_trustee,
+                                                            sdk_wallet_trust_anchor,
+                                                            sdk_pool_handle,
+                                                            build_revoc_def_by_default,
+                                                            claim_def, tconf):
+    revoc_def_req_trustee = send_revoc_reg_def(looper, txnPoolNodeSet, sdk_pool_handle, build_revoc_def_by_default,
+                                               claim_def, sdk_wallet_trustee)
+
+    rev_reg_entry = build_revoc_reg_entry_for_given_revoc_reg_def(revoc_def_req_trustee)
+    rev_reg_entry[VALUE][REVOKED] = [1, 2, 3, 4, 5]
+    del rev_reg_entry[VALUE][PREV_ACCUM]
+    rev_entry_req_trust_anchor = sdk_sign_request_from_dict(looper, sdk_wallet_trust_anchor, rev_reg_entry)
+    with pytest.raises(RequestRejectedException):
+        sdk_send_and_check([json.dumps(rev_entry_req_trust_anchor)], looper, txnPoolNodeSet, sdk_pool_handle)
+
+
+def test_trustee_not_owner_cant_create_revoc_reg_entry(looper,
+                                                       txnPoolNodeSet,
+                                                       sdk_wallet_trustee,
+                                                       sdk_wallet_trust_anchor,
+                                                       sdk_pool_handle,
+                                                       build_revoc_def_by_trust_anchor,
+                                                       claim_def, tconf):
+    revoc_def_req_trust_achor = send_revoc_reg_def(looper, txnPoolNodeSet, sdk_pool_handle,
+                                                   build_revoc_def_by_trust_anchor,
+                                                   claim_def, sdk_wallet_trust_anchor)
+
+    rev_reg_entry = build_revoc_reg_entry_for_given_revoc_reg_def(revoc_def_req_trust_achor)
     rev_reg_entry[VALUE][REVOKED] = [1, 2, 3, 4, 5]
     del rev_reg_entry[VALUE][PREV_ACCUM]
-    rev_entry_req = sdk_sign_request_from_dict(looper, sdk_wallet_client, rev_reg_entry)
-    sdk_send_and_check([json.dumps(rev_entry_req)], looper, txnPoolNodeSet, sdk_pool_handle)
\ No newline at end of file
+    rev_entry_req_trustee = sdk_sign_request_from_dict(looper, sdk_wallet_trustee, rev_reg_entry)
+    with pytest.raises(RequestRejectedException):
+        sdk_send_and_check([json.dumps(rev_entry_req_trustee)], looper, txnPoolNodeSet, sdk_pool_handle)
```

Comment:
But this is not an edit action. You created just one REVOC_REG_ENTRY. For the edit action, you need to create a txn (ADD action), and then try to modify it (EDIT action).